torch/quantization/quantize.py

```diff
@@ -229,6 +232,8 @@ def forward(self, x):
     nn.Linear: nnqd.Linear,
     nn.LSTM: nnqd.LSTM,
 }
+# List of modules for which we do not need FakeQuant for activations
+DEFAULT_QAT_SKIP_MODULE_LIST = [DeQuantStub]
```

Comment:
shall we do whitelist?

Reply:
This would need to be done for all the lists in quantize, so that the API is consistent. Shall we do this in a later PR?

Reply:
we can remove this